<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electrical Grid Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="layout_upper_toolbar">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_graph">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1367</width>
           <height>649</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Graph</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="layout_graph"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_tables">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1367</width>
           <height>649</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Data model</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="layout_data">
            <item>
             <widget class="QTabWidget" name="tab_data">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_bus">
               <attribute name="title">
                <string>Buses</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <layout class="QVBoxLayout" name="layout_bus"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_line">
               <attribute name="title">
                <string>Lines</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QGroupBox" name="groupBox_5">
                  <property name="title">
                   <string>AC lines</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <layout class="QVBoxLayout" name="layout_line"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_6">
                  <property name="title">
                   <string>DC lines</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <layout class="QVBoxLayout" name="layout_dcline"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_impedance">
               <attribute name="title">
                <string>Impedances</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <layout class="QVBoxLayout" name="layout_impedance"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>Transformers</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>Two winding transformers</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_6">
                   <item>
                    <layout class="QVBoxLayout" name="layout_trafo"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_4">
                  <property name="title">
                   <string>Three winding transformers</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <layout class="QVBoxLayout" name="layout_trafo3w"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Generators</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <layout class="QVBoxLayout" name="layout_gen"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_5">
               <attribute name="title">
                <string>Static Generators</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Symmetric static generators</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <layout class="QVBoxLayout" name="layout_sgen"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string>Asymmetric static generators</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                    <layout class="QVBoxLayout" name="layout_asgen"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_7">
               <attribute name="title">
                <string>External Grids</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <layout class="QVBoxLayout" name="layout_ext_grid"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>Symmetric Loads</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <layout class="QVBoxLayout" name="layout_load"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Other Loads</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QGroupBox" name="groupBox_7">
                  <property name="title">
                   <string>Asymmetric Loads</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_14">
                   <item>
                    <layout class="QVBoxLayout" name="layout_aload"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_8">
                  <property name="title">
                   <string>Motors</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_15">
                   <item>
                    <layout class="QVBoxLayout" name="layout_motor"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_6">
               <attribute name="title">
                <string>Shunts</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <layout class="QVBoxLayout" name="layout_shunt"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="title">
                <string>Wards</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QGroupBox" name="groupBox_9">
                  <property name="title">
                   <string>Wards</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_17">
                   <item>
                    <layout class="QVBoxLayout" name="layout_ward"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_10">
                  <property name="title">
                   <string>Extended wards</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_18">
                   <item>
                    <layout class="QVBoxLayout" name="layout_xward"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_9">
               <attribute name="title">
                <string>Storages</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_19">
                <item>
                 <layout class="QVBoxLayout" name="layout_storage"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_10">
               <attribute name="title">
                <string>Switches</string>
               </attribute>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item>
                 <layout class="QVBoxLayout" name="layout_switch"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
